web/src/portico/signup.js

```diff
@@ -284,4 +284,10 @@ $(() => {
         }
         $("#new-user-email-address-visibility .current-selected-option").text(selected_option_text);
     });
+
+    $("#registration").on("click", ".edit-realm-details", (e) => {
+        $("#registration .not-editable-realm-details").addClass("hide");
+        $("#registration .realm-creation-editable-inputs").removeClass("hide");
+        $(e.target).hide();
```

Comment:
Does this do something good with keyboard focus? I could imagine a `preventDefault` being useful as part of making that work reliably.

Reply:
I am not sure how adding `preventDefault` would help here.

Currently the focus is on "Full name" input, when the form is first opened. When user clicks on pencil icon, none of the inputs is focussed. We might need to focus it on the "Organization name" input in that case.

Also, the pencil icon is not focusable right now. We should make it focusable and might also want a handler to opend the editable form on pressing `Enter` when the icon is in focus.